### PR TITLE
TC 06.1.4_01 Checking that the grid is selected and has 12 positions by default

### PR DESCRIPTION
**Pull request by trukhachev495**
[Trello C 06.1.4_01](https://trello.com/c/brGJWiW8/266-tc-061401-checking-that-the-grid-is-selected-and-has-12-positions-by-defaulturl)

### CODE REVIEW

File: tests/menTopsDisplayMode.spec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('menTopsDisplayMode', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+  })
+test("Checking that the grid is selected and has 12 positions by default", async ({ page }) => {
+    await page.goto('https://magento.softwaretestingboard.com/')
```
Comment (S-Lanchik) [inline tests/menTopsDisplayMode.spec.js:8]: You don't need this line, line 5 does the same action.

Comment (annanazcz) [inline tests/menTopsDisplayMode.spec.js:8]: ты этот же степ уже делаешь на 4й строке ;)

Comment (S-Lanchik) [reply tests/menTopsDisplayMode.spec.js:8]: Add PR link to Trello

**Review by annanazcz — CHANGES_REQUESTED** [review]

File: tests/menTopsDisplayMode.spec.js (hunk 1 above)
Comment (trukhachev495) [reply tests/menTopsDisplayMode.spec.js:8]: забыл удалить, делал в другом файле, спасибо)

Comment (trukhachev495) [reply tests/menTopsDisplayMode.spec.js]: ok

**Review by verafes — APPROVED** [review]

**Review by Sensation3000 — APPROVED** [review]

File: tests/menTopsDisplayMode.spec.js (hunk 1 above)
Comment (Sensation3000) [reply tests/menTopsDisplayMode.spec.js]: @annanazcz please do not request changes for minor mistakes. Just write in the comments section about this without "requesting". Cause @trukhachev495 after that is waiting only your approval. The mistake has been corrected 6 hours  ago and PRs still waits for you to approve

File: tests/menTopsDisplayMode.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('menTopsDisplayMode', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+  })
+test("Checking that the grid is selected and has 12 positions by default", async ({ page }) => {
+    await page.goto('/')
```
Comment (Sensation3000) [inline tests/menTopsDisplayMode.spec.js:8]: the 5 and 8 line are doing the same thing

**Pull request closed without merging**